include/mbedtls/cmac.h

```diff
@@ -93,8 +94,9 @@ int mbedtls_cipher_cmac_starts( mbedtls_cipher_context_t *ctx,
  * \param input         The buffer holding the input data.
  * \param ilen          The length of the input data.
  *
- * \returns             \c 0 on success, or #MBEDTLS_ERR_MD_BAD_INPUT_DATA
- *                      if parameter verification fails.
+ * \returns            \c 0 on success.
+ * \returns            #MBEDTLS_ERR_MD_BAD_INPUT_DATA
+ *                     if parameter verification fails.
```

Comment:
Suggestion: Changeto `if the parameter verification fails.`

Reply:
No, it is correct the way it is.

Reply:
Ok, I'm not a native, I thought there'd need to be an article here, strictly speaking.